frost-core/src/frost.rs

```diff
@@ -148,34 +148,67 @@ where
     }
 }
 
-/// Generates the lagrange coefficient for the i'th participant.
-#[cfg_attr(feature = "internals", visibility::make(pub))]
-fn derive_interpolating_value<C: Ciphersuite>(
-    signer_id: &Identifier<C>,
-    signing_package: &SigningPackage<C>,
+/// Generates a lagrange coefficient.
+///
+/// The Lagrange polynomial for a set of points (x_j, y_j) for 0 <= j <= k
+/// is ∑_{i=0}^k y_i.ℓ_i(x), where ℓ_i(x) is the Lagrange basis polynomial:
+///
+/// ℓ_i(x) = ∏_{0≤j≤k; j≠i} (x - x_j) / (x_i - x_j).
+///
+/// This computes ℓ_j(x) for the set of points `xs` and for the j corresponding
+/// to the given xj.
+///
+/// If `x` is None, it uses 0 for it (since Identifiers can't be 0)
+pub fn compute_lagrange_coefficient<C: Ciphersuite>(
```

Comment:
<3

Reply:
I'm torn between these unicode chars that are good and doing this with LaTeX which we can render nicely in the rustdoc...

Reply:
Ah vanilla, yes, but we can pull in katex/mathjax in the html include in the header, which is a stable feature now, and docs.rs will render it, but it's a pro/con on the actual rustdoc text readability when inline with code

Reply:
(this is non-blocking, just a continual thing I go back and forth on with math in rustdoc :D)

Reply:
Ah got it, I wasn't aware `--html-in-header` was supported in docs.rs. I'm inclined to leave as is since the formulas aren't that complicated, but I can change it in a followup PR if you prefer to

Reply:
Also this made me realize the function is `pub` but it doesn't need to be, I made it optionally pub under `internals` feature in a3558ea